Tutorials/Add-Google-Authentication/index.md

```diff
@@ -20,17 +20,17 @@ Developers who won't mind configuring a Google API, running the solution in Visu
 
 ### Prerequisites
 Here is what you will need for this tutorial:
-- Visual studio installed
-- A Google account
-- A working Umbraco solution
+- [Visual Studio](https://visualstudio.microsoft.com/) installed
+- A [Google](https://www.google.com/intl/en-GB/gmail/about/#) account
+- A working [Umbraco Solution](https://umbraco.com/products/umbraco-for-enterprise-solutions/)
```

Comment:
The link does not make sense IMHO.  Wouldn't a link to the getting started section be better?  E.g. https://our.umbraco.com/documentation/Getting-Started/

Don't forget to make the link relative and not absolute.